.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
Right now this step doesn't do anything. Is there any build step that _can_ be done? If not, might make sense just to eliminate this part.

Reply:
There is nothing to build. Therefore I will remove this step